internal/email/email.handler.go

```diff
@@ -0,0 +1,79 @@
+package email
+
+import (
+	"github.com/brain-flowing-company/pprp-backend/apperror"
+	"github.com/brain-flowing-company/pprp-backend/config"
+	"github.com/brain-flowing-company/pprp-backend/internal/models"
+	"github.com/brain-flowing-company/pprp-backend/utils"
+	"github.com/gofiber/fiber/v2"
+	"go.uber.org/zap"
+)
+
+type Handler interface {
+	SendVerificationEmail(c *fiber.Ctx) error
+	VerifyEmail(c *fiber.Ctx) error
+}
+
+type handlerImpl struct {
+	service Service
+	logger  *zap.Logger
+	cfg     *config.Config
+}
+
+func NewHandler(logger *zap.Logger, cfg *config.Config, service Service) Handler {
+	return &handlerImpl{
+		service,
+		logger,
+		cfg,
+	}
+}
+
+type Body struct {
+	Email string `json:"email"`
+}
+
+func (h *handlerImpl) SendVerificationEmail(c *fiber.Ctx) error {
+	body := Body{}
+
```

Comment:
Since this model only uses once in this handler, you can move it inside the func.
ex.
```golang
body := struct{
  Email string `json:email"`
}{}
```
or
```golang
type Body struct{ Email string `json:email"` }
body := Body{}
```
etc.
Depends on your desire level of aesthetic.
With this way, the body struct doesn't get exposed to other packages.